.changeset/smart-apes-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/context': patch
+---
+
+Make property decorated with `@provide()` configurable. This allows chaining additional decorators that modify the property.
```

Comment:
Worth mentioning this is an experimental decorator only change? I assume standard decorators already work correctly.

Reply:
thank you, done!